include/ua_client_highlevel.h

```diff
@@ -589,8 +589,8 @@ UA_Client_Subscriptions_manuallySendPublishRequest(UA_Client *client);
 
 typedef void (*UA_MonitoredEventHandlingFunction)(UA_UInt32 monId,
                                                   size_t nEventFields,
-                                                  UA_Variant *eventFields,
-                                                  void *context);
+                                                  const UA_Variant *eventFields,
+                                                  const void *context);
 
```

Comment:
I would also vote to set non-pointer parameters to const, i.e., `const UA_UInt32 monId`.

See also the discussion here: https://stackoverflow.com/questions/2452132/does-it-ever-make-sense-to-make-a-fundamental-non-pointer-parameter-const